modules/deploy/pages/setting-up-dr-cluster.adoc

```diff
@@ -40,33 +41,67 @@ It serves an important role in supporting Disaster Recovery (DR) and Data Migrat
 
 == Recommended Deployment Models
 
+
+=== Zero Downtime Active-Active Disaster Recovery
+// tag::zero-downtime-active-active[]
+
+This model provides true zero-downtime disaster recovery using bi-directional XDCR between two active mobile clusters.
+Both clusters remain operational, with no fail-over through load balancer switching.
+You must configure both clusters with `shared_bucket_access=true` and `import_docs=true`.
```

Comment:
`shared_bucket_access=true` is required when loading a database in 4.0. https://jira.issues.couchbase.com/browse/CBG-4726 (ticket will not be public until the release happens).

Setting `shared_bucket_access=false` has been deprecated for a long time.